JPS_Ontology/ontology/ontoheatnetwork/ontoheatnetwork.owl

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0"?>
-<rdf:RDF xmlns="https://www.theworldavatar.com/kg/ontoheatnetwork/"
-     xml:base="https://www.theworldavatar.com/kg/ontoheatnetwork/"
-     xmlns:OntoHeatNetwork="https://www.theworldavatar.com/kg/ontoheatnetwork/"
+<rdf:RDF xmlns="http://theworldavatar.com/ontology/ontoheatnetwork/OntoHeatNetwork.owl#"
```

Comment:
Please use the new format, i.e. `https://www.theworldavatar.com/kg/ontoheatnetwork/`. This might need some manual string replacement as the TBox Generator (used to?) faces issues with namespaces not ending with `#`

Reply:
Addressed by [7da7121](https://github.com/cambridge-cares/TheWorldAvatar/pull/453/commits/7da712107b2765ea2d862b115a85bb9d3f330be7)